qiita_ware/ebi.py

```diff
@@ -20,6 +21,11 @@
 from qiita_db.util import convert_to_id
 
 
+class EmptyFileWarning(Warning):
```

Comment:
I recommend to add this warning in the exceptions class and add a filter so the Warning is always raised. There is an example in qiita_db.exceptions.